<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveCorrection</class>
 <widget class="QWidget" name="CurveCorrection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Gamma Correction</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCustomPlot" name="gamma_plot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Coefficient A:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Coefficient Gamma:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>2.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>S-Correction</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCustomPlot" name="s_plot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Middle Point:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="value">
                <number>127</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Coefficient A:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
               <property name="minimum">
                <double>-99.989999999999995</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Coefficient Gamma:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-99.989999999999995</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>2.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Custom Curve Correction</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QCustomPlot" name="custom_plot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Custom Points (x,y,x,y,x,y...):</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>47</height>
          </size>
         </property>
         <property name="plainText">
          <string>0 0.00 1 0.01 2 0.02 3 0.03 4 0.04 5 0.05 6 0.06 7 0.07 8 0.08 9 0.09 10 0.10 11 0.10 12 0.11 13 0.12 14 0.13 15 0.14 16 0.15 17 0.16 18 0.16 19 0.17 20 0.18 21 0.19 22 0.20 23 0.21 24 0.21 25 0.22 26 0.23 27 0.24 28 0.24 29 0.25 30 0.26 31 0.27 32 0.27 33 0.28 34 0.29 35 0.30 36 0.30 37 0.31 38 0.32 39 0.32 40 0.33 41 0.34 42 0.34 43 0.35 44 0.36 45 0.36 46 0.37 47 0.37 48 0.38 49 0.39 50 0.39 51 0.40 52 0.41 53 0.41 54 0.42 55 0.42 56 0.43 57 0.43 58 0.44 59 0.45 60 0.45 61 0.46 62 0.46 63 0.47 64 0.47 65 0.48 66 0.48 67 0.49 68 0.49 69 0.50 70 0.50 71 0.51 72 0.51 73 0.52 74 0.52 75 0.53 76 0.53 77 0.54 78 0.54 79 0.55 80 0.55 81 0.56 82 0.56 83 0.56 84 0.57 85 0.57 86 0.58 87 0.58 88 0.59 89 0.59 90 0.59 91 0.60 92 0.60 93 0.61 94 0.61 95 0.61 96 0.62 97 0.62 98 0.62 99 0.63 100 0.63 101 0.64 102 0.64 103 0.64 104 0.65 105 0.65 106 0.65 107 0.66 108 0.66 109 0.66 110 0.67 111 0.67 112 0.67 113 0.68 114 0.68 115 0.68 116 0.69 117 0.69 118 0.69 119 0.70 120 0.70 121 0.70 122 0.70 123 0.71 124 0.71 125 0.71 126 0.72 127 0.72 128 0.72 129 0.72 130 0.73 131 0.73 132 0.73 133 0.74 134 0.74 135 0.74 136 0.74 137 0.75 138 0.75 139 0.75 140 0.75 141 0.76 142 0.76 143 0.76 144 0.76 145 0.77 146 0.77 147 0.77 148 0.77 149 0.77 150 0.78 151 0.78 152 0.78 153 0.78 154 0.79 155 0.79 156 0.79 157 0.79 158 0.79 159 0.80 160 0.80 161 0.80 162 0.80 163 0.80 164 0.81 165 0.81 166 0.81 167 0.81 168 0.81 169 0.82 170 0.82 171 0.82 172 0.82 173 0.82 174 0.82 175 0.83 176 0.83 177 0.83 178 0.83 179 0.83 180 0.83 181 0.84 182 0.84 183 0.84 184 0.84 185 0.84 186 0.84 187 0.85 188 0.85 189 0.85 190 0.85 191 0.85 192 0.85 193 0.85 194 0.86 195 0.86 196 0.86 197 0.86 198 0.86 199 0.86 200 0.86 201 0.87 202 0.87 203 0.87 204 0.87 205 0.87 206 0.87 207 0.87 208 0.88 209 0.88 210 0.88 211 0.88 212 0.88 213 0.88 214 0.88 215 0.88 216 0.88 217 0.89 218 0.89 219 0.89 220 0.89 221 0.89 222 0.89 223 0.89 224 0.89 225 0.89 226 0.90 227 0.90 228 0.90 229 0.90 230 0.90 231 0.90 232 0.90 233 0.90 234 0.90 235 0.90 236 0.91 237 0.91 238 0.91 239 0.91 240 0.91 241 0.91 242 0.91 243 0.91 244 0.91 245 0.91 246 0.91 247 0.92 248 0.92 249 0.92 250 0.92 251 0.92 252 0.92 253 0.92 254 0.92 255 0.92 </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>libs/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
